samples/subsys/ipc/openamp/remote/src/main.c

```diff
@@ -35,8 +35,12 @@ static struct metal_device shm_device = {
 			.virt       = (void *) SHM_START_ADDR,
 			.physmap    = shm_physmap,
 			.size       = SHM_SIZE,
-			.page_shift = 0xffffffff,
-			.page_mask  = 0xffffffff,
+/*
+ * 32 bit: -1 == 0xffffffff;
+ * 64 bit: -1 == 0xffffffffffffffff;
+ */
+			.page_shift = (unsigned long)-1,
+			.page_mask  = (unsigned long)-1,
```

Comment:
Perhaps `-1UL` is better here.

For readability, I suggest you use macro rather than number. Refer to https://github.com/OpenAMP/libmetal/blob/23ba77941cf38a916d7a9fe5137a8bd2ee1e6122/examples/system/generic/zynqmp_r5/zynqmp_amp_demo/sys_init.c#L44-L45
```
#define DEFAULT_PAGE_SHIFT (-1UL)
#define DEFAULT_PAGE_MASK  (-1UL)
```

Reply:
fixed